### PR TITLE
Iconic display mode

### LINKED ISSUE #186: Definición de procedimientos y funciones primitivas con íconos
La idea es poder asignarle a nivel contenidista un ícono tanto a bloques nativos Gobstones (opcional) como procedimientos y funciones primitivas definidas por el usuario (fundamental). 

# :eyes: Prototipo

image

# :repeat: Opciones para la repetición simple

**Importante:** los colores de las capturas que hay debajo son los que por defecto tiene el componente de bloques, pero en la versión final serán los de Mumuki (como se ve en la captura de arriba). Observar solamente el estilo de encastre del número con el bloque repetir.

### Input inline, con íconos de 32px y 64px

Notar que el "agujero" nunca cambia de tamaño y termina quedando chico en comparación con el bloque que se va a encastrar allí. Una vez encastrado se expande, como se ve en la primera imagen. 

image image image

### Input externo, arriba, con íconos de 32px y 64 px

En este caso, lo que no cambia de tamaño es la muesca donde va la expresión numérica, que puede resultar muy pequeña en relación al tamaño del bloque. Cuando se encastra queda perfecto el tamaño.

image image image

### Input externo, arriba (no alineado)

image

### Input externo, abajo

image

### PR DESCRIPTION
**Pull request by faloi**
# :dart: Goal

This PR adds support for iconic blocks

# :warning:  Caveats

~Build will fail until https://github.com/mumuki/gs-element-blockly/pull/4 is merged and version `0.37.0` is released.~

# :memo: Details

This PR also moves all assets to the runner, and changes color for control structures.

# :back: Backward compatibility 

This PR is **not** backward compatible since color change has a huge impact on aesthetic. It  _may_ also break things because of the new assets serving strategy, and it will require a language re-import in laboratory. Please test with care. 

# :eyes: Relations

Closes #186

### CODE REVIEW

**Comment by flbulgarelli** [comment]
@faloi PS: I have formatted your PR message. Please review I have not misinterpreted those code changes.

**Comment by faloi** [comment]
> @faloi PS: I have formatted your PR message. Please review I have not misinterpreted those code changes.

Despite the usual typos, the new description is wonderful. :stuck_out_tongue:

File: lib/assets_server.rb [hunk 1]
```diff
@@ -29,15 +35,23 @@ def self.get_local_svg(name, asset_type)
...
   end
 
   ['attires_enabled', 'attires_disabled'].each do |name|
     get_local_svg(name, 'layout')
   end
+
+  get '/assets/editor/editor.html' do
+    cross_origin
+    @assets_url = "//#{request.host_with_port}/assets"
+    erb File.read('lib/render/editor/editor.html.erb')
```
Comment (luchotc) [inline lib/assets_server.rb:55]: Does this has a big impact on performance?

**Review by luchotc — COMMENTED** [review]
I'm sorry. Too little, too late. Although it's already merged I wanted to look at it anyway.
Looks great! That question only :smile:

File: lib/assets_server.rb (hunk 1 above)
Comment (faloi) [reply lib/assets_server.rb:55]: Don't really know how to measure it. :thinking:

**Pull request merged**